operations/finance.md

```diff
@@ -1485,7 +1485,7 @@ Adds a new external payment to a bill of the specified customer. An external pay
 | `CustomerId` | string | required | Unique identifier of the [Customer](customers.md#customer). |
 | `BillId` | string | optional | Unique identifier of an open bill of the customer where to assign the payment. |
 | `Amount` | [Amount value](finance.md#amount-value) | required | Amount of the external card payment. |
-| `Type` | string [External payment type](finance.md#external-payment-type) | optional | Type of the external payment. |
+| `Type` | string [External payment type](finance.md#external-payment-type) | optional | Type of the external payment.*Except for the properties based in the French Legal Environmen. Unspecified is considerer as fraud |
```

Comment:
- In the API docs, I think we use term enterprise, not property because property is also technical term.
- Missing space after first dot
- Typo in environmen**t**.
- Typo in considere**d**
- Missing dot after last sentence.

CC @vitasamek

Reply:
https://github.com/MewsSystems/gitbook-connector-api/commit/5b49a72dee65c5189bb78d1e184cfbf74dcc715d

Reply:
It wasn't merged to master. fixed in develop